WordPress/src/main/java/org/wordpress/android/GCMIntentService.java

```diff
@@ -115,6 +116,7 @@ protected void handleDefaultPush(Context context, Bundle extras) {
         }
 
         String iconUrl = extras.getString("icon");
+        String noteType = StringUtils.notNullStr(extras.getString("type"));
```

Comment:
We can use 'noteType' [here](https://github.com/wordpress-mobile/WordPress-Android/blob/feature/notifications-enhancements/WordPress/src/main/java/org/wordpress/android/GCMIntentService.java#L195) rather than read it again from the extras.